.goreleaser.yml

```diff
@@ -3,39 +3,40 @@
 before:
```

Comment:
What's the justification for the changes in this file? Where can I read about the "recommended pattern"?

Reply:
I linked to the TF provider docs in the PR description with a link to the template repo. Basically it adds the manifest and switches the YAML to standard formatting (double quotes and indentation for readability).

I've actually spotted a typo which I'm about to fix, when enabling `mod tidy`. I've re-enabled this as the criteria for taking it out looks to have passed based on the comment.